### ISSUE TITLE
Schedule and host WCAG 2.1 Training sessions

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#28175]
## Issue Description
On 7/29, Platform released the enforcement of WCAG 2.1 in Collaboration Cycle. This included the release of 2 slide decks:

1. New success criteria in WCAG 2.1
2. Foundational Testing for WCAG 2.1

In order to make sure VFS teams fully understand the WCAG changes, Platform would like to host at least 2 training sessions to record and share.

---
## Tasks
- [x] Determine @1Copenut availability since he will be leading the trainings
Monday, Aug. 16: 2pm ET to end of day
Tuesday, Aug. 17: 10-11am ET, and 2pm ET to end of day
Thursday, Aug. 19: 10am ET to 1pm ET
Friday, Aug. 20: 10am ET to 12 noon ET (except for 10:30 T3 standup)
   - [x] New success criteria date/time: 8/16 at 2:00pm ET
   - [x] Foundational testing date/time: 8/17 at 2:00pm ET
- [x] Schedule training sessions.
- [x] Post on Slack channels and announce training sessions at ToT
- [x] Host meeting.
   - [x] ensure auto-transcribe is enabled
   - [x] record meeting
  

## Acceptance Criteria
- [ ] _What will be created or happen as a result of this story?_
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by 1Copenut** [comment]
@shiragoodman Can we close this one?

**Comment by shiragoodman** [comment]
@1Copenut yes, but now we have [this one](https://github.com/department-of-veterans-affairs/va.gov-team/issues/28775) :)

**Comment by 1Copenut** [comment]
@shiragoodman 👍 Yep, that's my next goal, to write it all out as a document or two. I've got so many open threads I wanted to put one in the done column. :)
